pkg/plugins/localfiles.go

```diff
@@ -33,7 +33,7 @@ func NewLocalFS(basePath string) LocalFS {
 // fileIsAllowed takes an absolute path to a file and an os.FileInfo for that file, and it checks if access to that
 // file is allowed or not. Access to a file is allowed if the file is in the FS's Base() directory, and if it's a
 // symbolic link it should not end up outside the plugin's directory.
-func (f LocalFS) fileIsAllowed(basePath string, absolutePath string, info os.FileInfo) (bool, error) {
+func (f LocalFS) fileIsAllowed(basePath string, absolutePath string, info fs.FileInfo) (bool, error) {
```

Comment:
No big deal since

```go
package os 

// ...

type FileInfo = fs.FileInfo
```

But why was this changed?

Reply:
I don't recall my exact thinking at the time, but `fs.FileInfo` is the "more correct" one. I may have been thinking about refactoring this to use `io/fs` rather than `os` wherever I could for easier injection of in-memory filesystems at test-time.

However, given that this is the only change in this file, I'm going to back it out - it's not relevant to this PR so the change just confuses history.